src/Microsoft.SqlTools.ServiceLayer/TableDesigner/Contracts/Requests/ProcessTableDesignerEditRequest.cs

```diff
@@ -24,6 +24,12 @@ public class ProcessTableDesignerEditResponse
         public bool IsValid { get; set; }
 
         public TableDesignerIssue[] Issues { get; set; }
+
+        public bool IsEdge { get; set; }
```

Comment:
As mentioned in the other PR, I don't think we want to be including these MSSQL-specific properties in a generic API like this. Would suggest instead having a property bag of key/value pairs that store metadata like this (similar to how OE nodes have the metadata field)